package.json

```diff
@@ -55,5 +56,6 @@
   },
   "dependencies": {
     "@mapbox/tile-cover": "3.0.1"
-  }
+  },
+  "packageManager": "yarn@1.22.22"
```

Comment:
We are using volta in deck, could we be consistent here? https://github.com/visgl/deck.gl/blob/master/package.json#L73

Reply:
I'm a bit skeptical about mixing this with `deck.gl`, but if it helps you set up this environment, I'm more than happy to add it here, sure!